INSTALL.md

```diff
@@ -24,56 +20,142 @@
+ => [build 3/8] RUN mkdir -p /app/bin                                                                                                                44.2s
+ => [build 4/8] ADD https://github.com/teslamotors/vehicle-command/archive/refs/heads/main.zip /tmp                                                   9.0s
+ => [stage-1 3/5] RUN mkdir /data                                                                                                                    24.6s
+ => [build 5/8] RUN unzip /tmp/main.zip -d /app                                                                                                      23.0s
+ => [stage-1 4/5] COPY app /app                                                                                                                      11.8s
+ => [build 6/8] WORKDIR /app/vehicle-command-main                                                                                                     5.6s
+ => [build 7/8] RUN go get ./...                                                                                                                    162.6s
+ => [build 8/8] RUN go build -o /app/bin ./...                                                                                                     2438.3s
+ => [stage-1 5/5] COPY --from=build /app/bin/tesla-control /usr/bin/                                                                                 10.1s
+ => exporting to image                                                                                                                               16.1s
+ => => exporting layers                                                                                                                              15.8s
+ => => writing image sha256:8032d3fc0fb818244a987ea0691df0c666527e6ac2dafade1604dc37a97998e5                                                          0.1s
+```
+2. When completed, check it's there by issuing `docker images`. You will see somthing like this:
+```
+REPOSITORY                   TAG                 IMAGE ID       CREATED          SIZE
+tesla_ble_mqtt               latest              8032d3fc0fb8   14 minutes ago   34.1MB
+golang                       1.22.4-alpine3.20   ae51de63cd1f   3 weeks ago      227MB
+alpine                       3.20.0              31d13e3b0449   6 weeks ago      6.52MB
+hello-world                  latest              f0c407f2ecb9   4 months ago     8.95kB
+```
+3. Follow steps 1 & 2 exactly as per 'Deploy using Command Line'
+4. You will need to edit docker-compose.yml. Change the line `image: "iainbullock/tesla_ble_mqtt:latest"` to `image: "tesla_ble_mqtt:latest"`
+5. Follow steps 4 to 8 exactly as per 'Deploy using Command Line'
+#### Deploy using Dockerfile via Portainer ####
+Again this is for those who can't (or don't want to) use the pre-built images from Dockerhub
+TODO
 ### Activate the Key via Home Assistant

```

Comment:
I feel this is quite far below all the docker stuff. Maybe add numbering to make it clearer?
1. Deploy the container
2. Activate the key
3. Add the key to the vehicle
4. Play around

Reply:
I think I will keep 'Deploy using Command Line' in here as this is what most people do, but move (and link) 'Deploy using Portainer' and 'Deploy using Dockerfile via Command Line' into separate files. This will make the whole INSTALL.md shorter and easier to follow. Let see what it then looks like, and decide if further re-numbering is needed

Reply:
Done

Reply:
Well not the Portainer stuff I will do this on the next release as I don't have time right now